<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="networkBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">network: rgb(153, 255, 150);
color: rgb(148, 33, 147);
font: 16pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="text">
    <string>Start Network</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addnodebtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">network: rgb(153, 255, 150);
color: rgb(148, 33, 147);
font: 16pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="text">
    <string>Add New Node</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mineButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>161</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">network: rgb(153, 255, 150);
color: rgb(148, 33, 147);
font: 16pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="text">
    <string>Stop Network</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="newnodeid">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>250</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="node">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>298</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="removebutton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>161</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">network: rgb(153, 255, 150);
color: rgb(148, 33, 147);
font: 16pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="text">
    <string>Remove Node</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>420</y>
     <width>100</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(146, 144, 0);
color: rgb(148, 33, 147);
font: 16pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="opntrnxnBttn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>190</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">network: rgb(153, 255, 150);
color: rgb(148, 33, 147);
font: 16pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="text">
    <string>Open Transaction List</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="peer">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>350</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="peerbutton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">network: rgb(153, 255, 150);
color: rgb(148, 33, 147);
font: 16pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="text">
    <string>Peer Node List</string>
   </property>
  </widget>
  <widget class="QPushButton" name="channelBtn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">network: rgb(153, 255, 150);
color: rgb(148, 33, 147);
font: 16pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="text">
    <string>Join Channel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(148, 55, 255);
color: rgb(146, 144, 0);

font: 24pt &quot;.AppleSystemUIFont&quot;;
</string>
   </property>
   <property name="text">
    <string>Welcome to the Admin Portal</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
